src/ray/raylet/worker_pool.h

```diff
@@ -179,12 +184,12 @@ class WorkerPool {
   /// The number of workers per process.
   int num_workers_per_process_;
   /// Pool states per language.
-  std::unordered_map<Language, State> states_by_lang_;
+  std::unordered_map<Language, State, std::hash<int>> states_by_lang_;
 
  private:
   /// A helper function that returns the reference of the pool state
   /// for a given language.
-  State &GetStateForLanguage(const Language &language);
+  inline State &GetStateForLanguage(const Language &language);
```

Comment:
No need to use `inline` here unless one of the following cases:
1. define this method in `.h` file.
2. declare this method in `xxx.cc` and also use it only in `xxx.cc`.